modules/analysis-common/src/test/resources/rest-api-spec/test/analysis-common/40_token_filters.yml

```diff
@@ -568,3 +568,98 @@
             filter:    [my_truncate]
     - length: { tokens: 1 }
     - match:  { tokens.0.token: foo }
+
+---
+"pattern_capture":
```

Comment:
yes, these rest tests are new. It is the only we can be sure we wired up the factory correctly and the token filter is actually working.